bees/devrantbee/devrantbee.go

```diff
@@ -81,61 +83,30 @@ func (mod *DevrantBee) Action(action bees.Action) []bees.Placeholder {
 }
 
 func (mod *DevrantBee) triggerEvent(rant devgorant.RantModel) {
+	v := reflect.ValueOf(rant)
+	names := make([]string, v.NumField())
+	types := make([]string, v.NumField())
+	values := make([]interface{}, v.NumField())
+
+	opts := make([]bees.Placeholder, v.NumField())
+
+	for i := 0; i < v.NumField(); i++ {
+		names[i] = v.Type().Field(i).Name
+		types[i] = v.Field(i).Type().String()
+		values[i] = v.Field(i).Interface()
+
+		// Parsing the values into a bees.Placeholder struct
+		opts[i] = bees.Placeholder{
+			Name:  names[i],
+			Type:  types[i],
```

Comment:
This could be a bit problematic, as we have some internal types that don't map to a Golang-type (e.g. `address` comes to mind) and vice versa. We could probably tag those fields in the structs, though.